mojaloop/values.yaml

```diff
@@ -2881,6 +2881,122 @@ account-lookup-service:
 
     affinity: {}
 
+  als-oracle-pathfinder:
+    enabled: false
```

Comment:
Note that this flag will not work. I assume you have tested the deployment?

Please confirm if the als-oracle-pathfinder is being deployed even through `enabled: false` has been configured?

Reply:
@mdebarros it does not deploy when enable is set to false